src/glass/src/app/pages/welcome-page/welcome-page.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="glass-welcome-page"
+     fxLayout="column"
+     fxLayoutAlign="center center">
+  <div class="card">
+    <div class="card-header">
+      <div class="card-title"
+           translate>
+        Welcome to Aquarium
+      </div>
+    </div>
+    <div class="card-body">
+      <div class="card-text"
+           *ngIf="!checked"
+           translate>
+        In order to get this node started and ensure anything works as smoothly as possible, we need to check if it
```

Comment:
IMO adding the `to` makes the sentence easier to read.
```suggestion
        In order to get this node started and to ensure anything works as smoothly as possible, we need to check if it
```

Reply:
I asked @asettle if she can take a look at the text snippets.